nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/automation/SuggestTagEntry.java

```diff
@@ -18,8 +18,10 @@
  */
 package org.nuxeo.ecm.platform.tag.automation;
 
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
```

Comment:
`java.util.Set` is an unused import.
Also the `org.nuxeo.ecm.platform.tag.Tag` import can be removed (in this file and others were usage of `Tag` was removed)